src/proj_capi.jl

```diff
@@ -69,7 +69,7 @@ end
 
 "Get human readable error string from proj.4 error code"
 function _strerrno(code::Cint)
-    bytestring(ccall((:pj_strerrno, libproj), Cstring, (Cint,), code))
+    @compat unsafe_string(ccall((:pj_strerrno, libproj), Cstring, (Cint,), code))
```

Comment:
This is not a syntax change, so `@compat` should not be needed here - just `using Compat` should do the trick (From Compat's README it looks like we'll need at least Compat v0.8.0 in the REQUIRE to get `unsafe_string`).